app/api/events.py

```diff
@@ -104,6 +105,14 @@ class Meta:
                                  schema='EventInvoiceSchema',
                                  many=True,
                                  type_='event_invoice')
+    discount_codes = Relationship(attribute='discount_code',
```

Comment:
I followed it form here. Maybe I am wrong. Please explain this also:
In the event model, we have `track` but in the api schema of events we have `tracks`.